index.bs

```diff
@@ -7905,14 +7911,14 @@ Constructors</h4>
 			4. Sort the elements in <var>tracks</var> based on their <code>id</code>
 				attribute using lexicographic ordering on sequences of code unit
 				values.
-			5. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
-				object. <a href="#audionode-constructor-init">Initialize</a>
-				<var>node</var>.
-			6. Set an internal slot <dfn attribute for="MediaStreamAudioSourceNode">[[input track]]</dfn> on this
+			5. Let <var>o</var> be an empty dictionnary.
+			6. <a href="#audionode-constructor-init">Initialize the AudioNode</a>
+				<var>this</var>, with <var>c</var> and <var>o</var> as arguments.
```

Comment:
Here, I assume `options` can be passed directly to the "initialize" algorithm, as the mediaStream attribute should be set on the node?